### PR TITLE
Fill in the registry images listing view

### PR DESCRIPTION
**Pull request by stefwalter**
This follows the recent designs. No filtering yet, the panels have not properly been filled in yet.

### CODE REVIEW

**Comment by stefwalter** [comment]
screenshot from 2015-12-14 21-43-11

**Comment by stefwalter** [comment]
Depends on #3330

File: pkg/registry/scripts/images.js [hunk 1]
```diff
@@ -40,12 +40,88 @@
...
+
+            $scope.imagestreams = function() {
+                return select().kind("ImageStream");
+            };
+        }
+    ])
+
+    .factory("imageLoader", [
+        "kubeLoader",
+        "kubeSelect",
```
Comment (sub-mod) [inline pkg/registry/scripts/images.js]: kubeSelect is unused

File: pkg/registry/scripts/images.js [hunk 2]
```diff
@@ -40,12 +40,88 @@
...
+            }
+
+            return {
+                watch: function() {
+                    if (watching)
+                        return;
+
+                    /* Load images, but fallback to loading individually */
+                    watching = loader.watch("imagestreams");
+                    loader.watch("images").catch(function(response) {
```
Comment (sub-mod) [inline pkg/registry/scripts/images.js]: @stefwalter how does catch work ?

Comment (stefwalter) [inline pkg/registry/scripts/images.js]: ```catch(errorCallback)``` – shorthand for ```promise.then(null, errorCallback)```

https://docs.angularjs.org/api/ng/service/$q

File: pkg/registry/scripts/images.js (hunk 1 above)
Comment (stefwalter) [inline pkg/registry/scripts/images.js]: Thanks. Removed.

**Pull request closed without merging**